src/templates/en/2019/chapters/pwa.html

```diff
@@ -10,9 +10,7 @@
   - make changes to the markdown content directly (`src/content/<lang>/<year>/<chapter>.md`) because any changes to the chapter templates will be overwritten by the generation script
 #}-->
 
-{% set metadata = {"part_number":"II","chapter_number":11,"title":"Progressive
-Web Apps","authors":["tomayac"," jeffposnick"],"reviewers":["hyperpress","
-ahmadawais"]} %} {% block main %}
+{% set metadata = {"part_number":"II","chapter_number":11,"title":"Progressive Web Apps","authors":["tomayac"," jeffposnick"],"reviewers":["hyperpress"," ahmadawais"]} %} {% block main %}
```

Comment:
It seems there's still whitespace creeping in. For example, see `" jeffposnick"`.

Reply:
That's due to the manual front-matter array parsing. I've added a trim, so that extra whitespace doesn't happen. (Sorry!)